includes/model/class-activity.php

```diff
@@ -148,8 +148,8 @@ public function from_post( Post $post ) {
 			$this->published = $object['published'];
 		}
 
-		$url = sprintf( '/%/users/%d/followers', ACTIVITYPUB_REST_NAMESPACE, intval( $post->get_post_author() ) );
-		$this->add_to( \get_rest_url( null, $url ) );
+		$path = sprintf( 'users/%d/followers', intval( $post->get_post_author() ) );
+		$this->add_to( get_rest_url_by_path( $path ) );
```

Comment:
`get_rest_url_by_path` is in the `Activitypub` namespace, so you have to add `use function Activitypub\get_rest_url_by_path` as the top of the class.

Reply:
Aha! I am not accustomed to thinking in namespaces :)